components/AlertNotification/AlertNotification.tsx

```diff
@@ -5,9 +5,11 @@ import { Alert as AlertDefault } from '../Alert/Alert';
 import { AlertTitle } from '../AlertTItle/AlertTitle';
 import { AlertDescription } from '../AlertDescription/AlertDescription';
 import { JSX } from 'react';
-import { CheckCircle, XCircle, Info, AlertTriangle } from 'lucide-react';
+import { CheckCircle, XCircle, Info, AlertTriangle, X } from 'lucide-react';
 import { IAlertNotification } from './AlertNotification.interface';
 import { AlertVariants } from './Alerts.enum';
+import { Button } from '../Button/Button';
+import toast from 'react-hot-toast';
 
```

Comment:
nit: please sort these alphabetically.

I'm aware these weren't sorted alphabetically prior to your work in the file, but we try and do so for ease of scanning the imports.

Reply:
@ryandotfurrer done